api_specs/v2/component_analyses.yaml

```diff
@@ -8,10 +8,78 @@ servers:
 security:
 - tokenAuth: []
 paths:
+  /component-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get security and other details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
+          in: header
+          description: User Identification uuid
+          required: false
+          schema:
+            type: string
+      requestBody:
+        description: CA Batch Request
+        required: true
+        content:
+          application/json:
+            schema:
```

Comment:
Define a Schema for request model.

Reply:
Its a schema definition only. What exactly do you want..?

Reply:
I was asking to move out schema definition to the schema section.

Something like https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/openapi.yaml#L35

Reply:
Ack!